deployments/rialto/docker-compose.yml

```
@@ -215,3 +215,9 @@ services:
       - "4567:4567"
     depends_on:
       - grafana-dashboard
+  front-end:
+    build:
+      context: .
+      dockerfile: ./Front-end.Dockerfile
+    ports:
+      - "80:80"
```

```suggestion
      - "8080:80"
```
Otherwise we will need `sudo` to run it.